domain-model-assistant/Packages/manifest.json

```diff
@@ -1,20 +1,22 @@
 {
   "dependencies": {
-    "com.unity.2d.animation": "7.0.3",
+    "com.unity.2d.animation": "8.0.3",
```

Comment:
The updates in the file come from Unity version updates and by using the update manager within Unity